server/channels/store/sqlstore/channel_store.go

```diff
@@ -1922,6 +1922,88 @@ func (s SqlChannelStore) UpdateMemberNotifyProps(channelID, userID string, props
 	return dbMember.ToModel(), err
 }
 
+// PatchMultipleMembersNotifyProps updates the NotifyProps of multiple channel members at once without modifying
+// unspecified fields. Like UpdateMemberNotifyProps, it doesn't modify the LastUpdateAt of affected rows.
```

Comment:
> it doesn't modify the LastUpdateAt of affected rows.

What's the backstory here? (This feels weird.)

Reply:
I don't know if it was an intentional choice or not. Based on the git history, and I don't remember it seeming like an intentional choice, so I decided to document it instead of having the behaviour of these methods diverge. Since the new API isn't just a batched version of `UpdateMemberNotifyProps` like my original version was, I'd be up to fix it for this case if we wanted. 0/5

Reply:
Looking back at the history of `UpdateMemberNotifyProps`, it looks like this was an unintentional change introduced by https://github.com/mattermost/mattermost/pull/18114. Previously, notify props were updated using the generic `UpdateMember` which calls `member.PreUpdate` which in turn updates `member.LastUpdateAt`.

I'll fix it for the new API, but I'm going to make a separate ticket to fix it for `UpdateMemberNotifyProps`. This new API is needed for 9.5, and I don't want to delay it to fix an existing API that isn't used by it